### DISCUSSION TITLE
Kernel Requirements - podman versions

### DISCUSSION THREAD

**Discussion by koteswara-daliparthi** [containers/podman#12840] [General]
Hello All,

I am Koti, i spent lot of hours to research the kernel requirements for podman versions. But none of my tries were fruitful. i would like some senior member to include the kernel requirements for each podman version. It would not only help newbies like me but also senior folk for reference.

Thanks 
Koti

**Comment by mheon** [comment]
We're very liberal about kernel version, and while we can take advantage of newer kernel features I'm not aware of any case where we've actually required them. I imagine any 4.x or newer kernel would work fine? I don't think we do any standardized testing on this, though.

**Reply by koteswara-daliparthi** [reply]
Hello Matt,
Thanks a lot for your response.  I am running an old kernel  2.6.36.4, so i am confused if i could build the newest podman, in my old kernel. That's why i thought of asking this question and request a wiki for the same.

Thanks
Koti

**Comment by rhatdan** [comment]
That might be a little old. Some versions of Podman run on RHEL7/Centos7.  But we no longer support those.  We are focused more on RHEL8/Centos8 Era kernels and newer.  That is the way I think about it.
Podman will likely work on older kernels as Root, but rootless mode will likely have a lot of issues.
